src/hooks/screen/useChatScreenHook.ts

```diff
@@ -123,6 +123,25 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
     }
   };
 
+  const handleChatData = async () => {
+    if (!localDb || !selectedChannel) return;
+    if (!otherListener[`chat_${selectedChannel?.id}`]) return;
+
+    try {
+      getAllMessages.refetch();
+      if (type === 'ANONYMOUS') {
+        const userInfo = await UserSchema.getSelfAnonUserInfo(
+          localDb,
+          anonProfileId,
+          selectedChannel?.id
+        );
+        setSelfAnonUserInfo(userInfo);
+      }
+    } catch (e) {
+      console.log(e, 'error get all chat');
+    }
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `handleChatData` function is missing error handling. If an error occurs during the execution of this function, it will be logged to the console but the application will continue running as if nothing happened. This could lead to unexpected behavior. Consider adding error handling logic to manage these situations more gracefully.
```diff
  const handleChatData = async () => {
    if (!localDb || !selectedChannel) return;
    if (!otherListener[`chat_${selectedChannel?.id}`]) return;

    try {
      getAllMessages.refetch();
      if (type === 'ANONYMOUS') {
        const userInfo = await UserSchema.getSelfAnonUserInfo(
          localDb,
          anonProfileId,
          selectedChannel?.id
        );
        setSelfAnonUserInfo(userInfo);
      }
    } catch (e) {
      console.log(e, 'error get all chat');
+     // Add error handling logic here
    }
  };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->